mmv1/third_party/terraform/services/compute/resource_compute_instance_template.go.erb

```diff
@@ -916,6 +917,19 @@ be from 0 to 999,999,999 inclusive.`,
 								If SEV_SNP, min_cpu_platform = "AMD Milan" is currently required.`,
 							AtLeastOneOf: []string{"confidential_instance_config.0.enable_confidential_compute", "confidential_instance_config.0.confidential_instance_type"},
 						},
+						<% else -%>
+						"confidential_instance_type": {
```

Comment:
Are we doing this if/else on the version only to change the description? You could just note in the description that TDX is only supported in beta or something like that and publish it to both providers. It would be cleaner than if/else on the whole block

Reply:
I agree that would look cleaner. I wonder if it would be confusing for the users to see the TDX option in the description at all.

Reply:
@slevenick Is this the preferred change?